### PR TITLE
new contact detail design

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
blue style for toolbar

__Tasks:__
- [x] contact detail according to new design
- [x] got feedback

__Depending PR:__
- https://github.com/massiveart/husky/pull/522

__Informations:__

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tests pass?   | none
| Fixed tickets | none
| Doc           | none

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/public/css/main.css [hunk 1]
```diff
@@ -58,6 +58,41 @@ input, select {
   background: #ccc;
 }
 
+/* line 60, ../scss/modules/base.scss */
+.highlight-section {
+  width: 100%;
+  background: #ddd;
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/public/css/main.css]: maybe use "$grayLighterer" which is #ddd ?

File: src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/edit/details/main.js [hunk 2]
```diff
@@ -423,12 +432,12 @@ define([
...
          */
         updateBankAccountAddIcon: function(numberOfBankAccounts) {
-            var $addIcon = this.sandbox.dom.find(constants.bankAccountAddSelector, this.$el),
+            var $addIcon = this.$find(constants.bankAccountAddId),
                 addIcon;
 
             if (!!numberOfBankAccounts && numberOfBankAccounts > 0 && $addIcon.length === 0) {
                 addIcon = this.sandbox.dom.createElement(customTemplates.addBankAccountsIcon);
-                this.sandbox.dom.after(this.sandbox.dom.find(constants.bankAccountsId), addIcon);
+                this.sandbox.dom.after(this.sandbox.dom.find('#bankAccounts'), addIcon);
```
Comment (nnatter) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/edit/details/main.js]: @marcelmoosbrugger maybe use "this.$find"

File: src/Sulu/Bundle/ContactBundle/Resources/public/js/components/contacts/edit/details/main.js [hunk 3]
```diff
@@ -461,17 +475,9 @@ define([
...
-                '.changeListener input, ' +
-                '.changeListener textarea');
-
-                this.sandbox.dom.on('#contact-form', 'keyup', function() {
+                this.sandbox.dom.on('#contact-form', 'change keyup', function() {
                     this.sandbox.emit('sulu.tab.dirty');
-                }.bind(this), '.changeListener select, ' +
-                '.changeListener input, ' +
-                '.changeListener textarea');
+                }.bind(this), 'select, ' + 'input, ' + 'textarea');
```
Comment (nnatter) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/components/contacts/edit/details/main.js]: @marcelmoosbrugger is there a reason why you don't use "'select, input, textarea'"?

File: src/Sulu/Bundle/AdminBundle/Resources/public/css/main.css (hunk 1 above)
Comment (marcelmoosbrugger) [inline src/Sulu/Bundle/AdminBundle/Resources/public/css/main.css]: this is the css

**Pull request merged**